app/presenters/vendor_api/application_presenter.rb

```diff
@@ -110,5 +110,9 @@ def domicile
 
       application_form.domicile
     end
+
+    def country
+      application_form.country[0..1]
```

Comment:
checking prod country can be `nil`, could we account for that here please?

Reply:
Oh wow, I didn't think that was possible, will amend.

Reply:
@cwrw https://github.com/DFE-Digital/apply-for-teacher-training/pull/6948/commits/8a4b68371f2c938f5de4147536d06b1fd3f94078